specs/validator/tableland-openapi-spec.yaml

```diff
@@ -122,52 +122,29 @@ paths:
                 $ref: "#/components/schemas/TxnReceipt"
         "400":
           description: Invalid query/statement value
-  /tables/structure/{hash}:
+  /tables/{chainId}/{tableId}:
     get:
       tags:
         - tables
-      summary: Finds tables by structure
-      description: Returns all tables that match a given structure
-      operationId: findTablesByStructure
+      summary: Get information about a table
+      description: Returns information about a single table, including schema information
+      operationId: getTableById
       parameters:
-        - name: hash
+        - name: chainId
           in: path
-          description: Structure hash to filter on
+          description: The parent chain to target
           required: true
-          schema:
-            type: string
-        - name: chainId
-          in: query
-          description: The parent chain to filter on
-          required: false
           schema:
             type: integer
             format: int32
-      responses:
-        "200":
-          description: successful operation
-          content:
-            application/json:
-              schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/Info"
-        "400":
-          description: Invalid structure value
-  /tables/{tableName}:
-    get:
-      tags:
-        - tables
-      summary: Get information about a table
-      description: Returns information about a single table, including schema information
-      operationId: getTableByName
-      parameters:
-        - name: tableName
+        - name: tableId
           in: path
-          description: Full name of table to return
+          description: Table identifier in the Registry.
```

Comment:
Similarly, I'm not entirely sure my use of "Registry" is fine here. I mean, it's correct, but maybe we want to avoid adding jargon. Again, no need to figure out this now but it can also be something to think about when polishing descriptions after.